npm/vue/src/index.ts

```diff
@@ -92,6 +92,7 @@ const cleanup = () => {
  * `MountOptions` are modifying, including some Cypress specific options like `styles`.
```

Comment:
I just followed the instructions above here (line 85-94) to fix it. Just copy paste the latest types.